distribution/lib/Standard/Test/0.0.0-dev/src/Bench.enso

```diff
@@ -230,11 +230,14 @@ type Bench
         duration_ns = conf.iterations * conf.seconds * 1000000000
         phase_start = System.nano_time
         stop_ns = phase_start + duration_ns
-        go = durations -> cur_ns ->
-            if cur_ns  > stop_ns then durations else
+        durations_builder = Vector.new_builder
+        go cur_ns =
+            if cur_ns  > stop_ns then Nothing else
                 dur = Bench.single_call act
-                @Tail_Call go (durations + [dur]) (cur_ns + dur)
```

Comment:
The old approach was doing far too much copying when concatenating the vectors - for benchmarks that are very quick and can do hundreds of thousands iterations in a second this was causing a huge slowdown of the benchmark run-time.